library/pluginsystem/PluginSystem.php

```diff
@@ -76,7 +76,7 @@ public function init()
         $composerJsonPath = realpath( __DIR__."/../../composer.json" );
         $composerJson = file_get_contents( $composerJsonPath );
         if ( $composerJson === false ) {
-            throw new Exception( "$composerJsonPath/composer.json is not readable" );
+            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
```

Comment:
Global namespace. 

Sent from my iPhone

> On Sep 5, 2016, at 12:38 PM, Art Eaton <notifications@github.com> wrote:
> 
> In library/pluginsystem/PluginSystem.php:
> 
> > @@ -76,7 +76,7 @@ public function init()
> >          $composerJsonPath = realpath( __DIR__."/../../composer.json" );
> >          $composerJson = file_get_contents( $composerJsonPath );
> >          if ( $composerJson === false ) {
> > -            throw new Exception( "$composerJsonPath/composer.json is not readable" );
> > +            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
> EDIT: I thought this was a goofup, until I realized that PHP must have stupidly decided to use a single backslash as the operator for a namespace....
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.